Translations/translation_EN.json

```diff
@@ -159,49 +159,49 @@
 				"Boost",
 				"temp"
 			],
-			"desc": "Tip temperature used in \"boost mode\""
+			"desc": "Tip temperature used in \"Boost mode\""
 		},
 		"AutoStart": {
 			"text2": [
 				"Start-up",
 				"behavior"
 			],
-			"desc": "O=off | S=heat up to soldering temp | Z=standby at sleep temp until moved | R=standby without heating until moved"
+			"desc": "O=off | S=heat up to soldering temp | Z=standby at sleep temp until moved | R=standby at no heat until moved"

```

Comment:
You can not say *'at'* no heat.
It has to be *'with'* no heat.
Which leads again to *'without'*, since this is the meaning of *'with no'*!